shell/i18n/fi.i18n.json

```diff
@@ -710,12 +710,12 @@
             "subtext": "Käyttäjät joiden sähköpostiosoitteena on tämä domain tulevat olemaan tämän palvelimen organisaation jäseniä."
           },
           "gappsToggle": {
-            "label": "Käyttäjät jotka tunnistautuvat G Suiten kautta",
+            "label": "Käyttäjät jotka tunnistautuvat Google Workspace kautta",
```

Comment:
@xet7, what's the right translation here?

I wonder how much effort we should put into finding the other translators vs. just letting these get a bit stale. For ones where we can't track down the translators, maybe it's better to use the old names than use the new name in the wrong language?

Reply:
My understanding (although I imagine it can be language-specific) is that generally brand names should not be translated, which is largely backed up by the contents of our current translations. (I also did run this string through Bing Translate and got a sane result, though that isn't authoritative.) I do think people today may not have any idea what Google Apps for Work is, and "G Suite" may not only suffer that, but may not obviously refer to Google.

I think in cases where we do have translated text, we should do our best to use Translate and similar tools to get close to accurate, and be very open to PRs that adjust i18n strings.